packages/engine/src/avatar/AvatarClientModule.ts

```diff
@@ -14,60 +14,60 @@ import FlyControlSystem from './FlyControlSystem'
 export function AvatarClientModule() {
   return [
     {
-      uuid: 'xre.engine.AvatarInputSystem',
+      uuid: 'ee.engine.AvatarInputSystem',
```

Comment:
oh lol, could you please revert all these, they will conflict with the system refactor i'm doing that also includes these changes